src/pages/workspace/workflows/WorkspaceWorkflowsPage.tsx

```diff
@@ -51,8 +54,32 @@ function WorkspaceWorkflowsPage({policy, betas, route}: WorkspaceWorkflowsPagePr
 
     const onPressAutoReportingFrequency = useCallback(() => Navigation.navigate(ROUTES.WORKSPACE_WORKFLOWS_AUTOREPORTING_FREQUENCY.getRoute(policy?.id ?? '')), [policy?.id]);
 
-    const items: ToggleSettingOptionRowProps[] = useMemo(
-        () => [
+    const authorizedPayerEmail = policy?.reimburserEmail;
+    const displayNameForAuthorizedPayer = useMemo(
```

Comment:
I think we'll have to refactor this to get the `reimburserAccountID` as well, so we don't need to do this email lookup. This can be a follow up though, since I created [this issue](https://github.com/Expensify/Expensify/issues/374749) for the approver flow as well. @lakchote